Explorer/Assets/DCL/Character/CharacterMotion/Systems/TeleportCharacterSystem.cs

```diff
@@ -0,0 +1,146 @@
+﻿using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Cysharp.Threading.Tasks;
+using DCL.AsyncLoadReporting;
+using DCL.CharacterCamera;
+using DCL.CharacterMotion.Components;
+using DCL.Diagnostics;
+using ECS.Abstract;
+using ECS.Prioritization;
+using ECS.Prioritization.Components;
+using ECS.SceneLifeCycle.Reporting;
+using System;
+using UnityEngine;
+
+namespace DCL.CharacterMotion.Systems
+{
+    /// <summary>
+    ///     Mutually exclusive with <see cref="InterpolateCharacterSystem" />. <br />
+    ///     This system reacts on the status of <see cref="AsyncLoadProcessReport" /> to make the appropriate changes in ECS
+    /// </summary>
+    [UpdateInGroup(typeof(ChangeCharacterPositionGroup))]
```

Comment:
Does this need a log group?

Reply:
It inherits a log category from the group